src/js/components/Menu.js

```diff
@@ -149,6 +160,13 @@ var MenuDrop = React.createClass({
     if (this.props.small) {
       classes.push(CLASS_ROOT + "__drop--small");
     }
+    if (this.props.className) {
+      var classNames = this.props.className.split(" ");
+      classNames = classNames.map(function(className) {
+        return className + "__drop";
```

Comment:
Since there could potentially be multiple drop menus on the same page which may need to be styled differently, the `*__drop` classes allow the developer to target individual drop menus independently.